tools/deploymentsSchemaTests/DeploymentsTests.cs

```diff
@@ -0,0 +1,25 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace deploymentsSchemaTests
+{
+    [TestClass]
+    public class DeploymentsTests
+    {
+        [TestMethod]
+        public void TestSchemaLoader()
+        {
+            // Calling Path.Combine to construct the path to schemas folder
+            var schemasPath = Path.Combine(new string[] { "..", "..", "..", "..", "..", "schemas" });
```

Comment:
>new string[] { "..", "..", "..", "..", "..", "schemas" } [](start = 43, length = 56)

@anthony-c-martin is there a better way to construct a path? relying on the number of parents seems a little fragile, although, is unlikely to change ...

Reply:
IMO declaring the dependency in the .csproj is more robust - e.g. something like:
```
<ItemGroup>
  <None Include="../../schemas/**" CopyToOutputDirectory="PreserveNewest" />
</ItemGroup>
```

That'll copy the files to the output directory at build time, so they should end up in the same dir as the DLLs, and you can rely on the path to reach them not changing. It should also fail the build if the path is incorrect.

Reply:
Note - you may also need to add a `[DeploymentItem]` attribute to your tests to register the schema files as test dependencies.